docs/development/writingzeppelininterpreter.md

```diff
@@ -22,6 +22,16 @@ Creating a new interpreter is quite simple. Just extend [org.apache.zeppelin.int
 
 You can include org.apache.zeppelin:zeppelin-interpreter:[VERSION] artifact in your build system.
 
+Your interpreter name is derived from the static register method
+
+```
+static {
+    Interpreter.register("MyInterpreterName", MyClassName.class.getName());
+  }
+```
+
+This name will appear later in the interpreter name option box during the interpreter configuration process.
+
```

Comment:
The name of the interpreter is what you later write to identify a paragraph which should be interpreted using this interpreter.

```
%MyInterpreterName
some interpreter spesific code...
```